environment.yml

```diff
@@ -1,15 +1,34 @@
-# You can use this file to create a conda environment for this pipeline:
-#   conda env create -f environment.yml
 name: nf-core-dualrnaseq-1.0dev
+
 channels:
   - conda-forge
   - bioconda
+  - anaconda
+  - agbiome
   - defaults
+
 dependencies:
-  - conda-forge::python=3.7.3
+  - bioconda::fastqc=0.11.9
+  - bioconda::samtools=1.9
+  - bioconda::salmon=1.3.0
+  - bioconda::STAR=2.7.3a
+  - bioconda::gffread=0.12.1
+  - bioconda::multiqc=1.8
+  - bioconda::htseq=0.12.4
+  - bioconda::bioconductor-rtracklayer=1.48.0
+  - bioconda::bioconductor-tximport=1.16.0
+  - conda-forge::python=3.7.6
+  - conda-forge::r-plyr=1.8.6
   - conda-forge::markdown=3.1.1
   - conda-forge::pymdown-extensions=6.0
-  - conda-forge::pygments=2.5.2
-  # TODO nf-core: Add required software dependencies here
-  - bioconda::fastqc=0.11.8
-  - bioconda::multiqc=1.7
+  - agbiome::bbtools=37.62
```

Comment:
bbtools is available from bioconda too - maybe worth to check whether you could get rid of the extra channel as this will slow down condas package resolving step 👍🏼 

There is no bbtools package I think, but bbmap, bbduk, ... are available. Depends a bit which ones you need.

Reply:
Been looking into this again. I remember when we added BBduk initially and I was surprised that it wasn't available on bioconda (same thought process as you). I then (and now) checked if BBduk is available as a standalone, and it's not. The only tool of the BBtools package we need is BBduk, so it looks like we will have to install this way.
I tried installing in an new environment locally and searched under https://anaconda.org
Is that where you would look?

Reply:
Ignore this - just seen your comment below and tested - nice find :-)